### PR TITLE
History Screen Prototype

### PR DESCRIPTION
**Pull request by at-haivo**
create history screen

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Youtube/Controllers/Base/BaseViewController.swift [hunk 1]
```diff
@@ -47,6 +47,16 @@ class BaseViewController: UIViewController {
         present(alertController, animated: true)
     }
 
+    // MARK: - Functions showAlert
+    func showAlert(message: String, okHandler: @escaping (UIAlertAction)-> Void) {
```
Comment (at-daiho) [inline Youtube/Controllers/Base/BaseViewController.swift]: WARNING - Return arrow and return type should be separated by a single space or on a separate line.

File: Youtube/Controllers/FavoriteVC/FavoriteViewController.swift [hunk 2]
```diff
@@ -34,21 +34,12 @@ class FavoriteViewController: BaseViewController {
     }
 
     func deleteAllButtonTouchUpInside(_ sender: UIButton) {
-        showAlert(message: "do you want delete all?")
-    }
-
-    func showAlert(message: String) {
-        let alertController = UIAlertController(title: Strings.youtubeTitle, message: message, preferredStyle: .alert)
-        let okAction = UIAlertAction(title: "OK", style: .default) { ( _ ) in
```
Comment (at-daiho) [inline Youtube/Controllers/FavoriteVC/FavoriteViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Youtube/Controllers/CategoryVC/DetailVC/DetailViewController.swift [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        tableView.register(HeaderTableView.self)
+        tableView.register(VideoTableViewCell.self)
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.backgroundColor = Color.black38
+
+        videos = createVideos()
+    }
+
+    private func createVideos() -> [Video] {
```
Comment (at-daiho) [inline Youtube/Controllers/CategoryVC/DetailVC/DetailViewController.swift]: Please add TODO

File: Youtube/Controllers/CategoryVC/DetailVC/DetailViewController.swift [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            videos.append(video)
+        }
+        return videos
+    }
+
+    func getVideo(videoCode: String) {
+        guard let url = URL(string: "https://www.youtube.com/embed/\(videoCode)") else {
+            return
+        }
+        myWebView?.loadRequest(URLRequest(url: url))
```
Comment (at-daiho) [inline Youtube/Controllers/CategoryVC/DetailVC/DetailViewController.swift]: Why you add ?

Comment (at-haivo) [inline Youtube/Controllers/CategoryVC/DetailVC/DetailViewController.swift]: add review demo

Comment (at-daiho) [inline Youtube/Controllers/CategoryVC/DetailVC/DetailViewController.swift]: Please add todo dummy data for all demo data

**Pull request merged**